From: Andrey Galkin <andrey@futoin.org>
Date: Thu, 21 Jan 2016 05:59:45 +0200
Subject: Applied the original debianization-fireqos.patch idea and updated to
 actual FireHOL v3

Description: debianization -- FireQOS
 Make /etc/fireqos the configuration folder for FireQOS;
 see debian/fireqos.README.Debian for details.
Author: Jerome Benoit <calculus@rezozer.net>
---
 doc/fireqos/fireqos-conf.5.md |  2 +-
 doc/fireqos/fireqos.1.md      |  4 ++--
 sbin/fireqos.in               | 10 +++++++++-
 3 files changed, 12 insertions(+), 4 deletions(-)

diff --git a/doc/fireqos/fireqos-conf.5.md b/doc/fireqos/fireqos-conf.5.md
index 463c6c7..2a0c993 100644
--- a/doc/fireqos/fireqos-conf.5.md
+++ b/doc/fireqos/fireqos-conf.5.md
@@ -15,7 +15,7 @@ extra-manpage: fireqos.conf.5
 This file defines the traffic shaping that will be applied by
 [fireqos(1)][].
 
-The default configuration file is `/etc/firehol/fireqos.conf`. It can be
+The default configuration file is `/etc/fireqos/fireqos.conf`. It can be
 overridden from the command line.
 
 A configuration consists of a number of input and output `interface`
diff --git a/doc/fireqos/fireqos.1.md b/doc/fireqos/fireqos.1.md
index ea71b12..7689a4c 100644
--- a/doc/fireqos/fireqos.1.md
+++ b/doc/fireqos/fireqos.1.md
@@ -25,7 +25,7 @@ FireQOS is a program that helps you configure traffic shaping on Linux.
 Run without any arguments, `fireqos` will present some help on usage.
 
 When given *CONFIGFILE*, `fireqos` will use the named file instead of
-`/etc/firehol/fireqos.conf` as its configuration.
+`/etc/fireqos/fireqos.conf` as its configuration.
 
 The parameter *name* always refers to an interface name from the
 configuration file. The parameter *class* always refers to a named class
@@ -96,7 +96,7 @@ requeues
 
 # FILES
 
-`/etc/firehol/fireqos.conf`
+`/etc/fireqos/fireqos.conf`
 
 # SEE ALSO
 
diff --git a/sbin/fireqos.in b/sbin/fireqos.in
index 5e2659c..e06f446 100755
--- a/sbin/fireqos.in
+++ b/sbin/fireqos.in
@@ -50,6 +50,7 @@ do
 done
 
 FIREHOL_CONFIG_DIR="$sysconfdir_POST/firehol"
+FIREQOS_CONFIG_DIR="$sysconfdir_POST/fireqos"
 common_disable_localization || exit
 common_public_umask || exit
 common_require_root || exit
@@ -62,7 +63,7 @@ export PATH="$PATH:/sbin:/usr/sbin:/usr/local/sbin"
 shopt -s extglob
 
 FIREQOS_SYSLOG_FACILITY="daemon"
-FIREQOS_CONFIG="${FIREHOL_CONFIG_DIR}/fireqos.conf"
+FIREQOS_CONFIG="${FIREQOS_CONFIG_DIR}/fireqos.conf"
 FIREQOS_LOCK_FILE=/var/run/fireqos.lock
 FIREQOS_LOCK_FILE_TIMEOUT=600
 FIREQOS_DIR=/var/run/fireqos
@@ -111,6 +112,13 @@ then
 	source "${FIREHOL_CONFIG_DIR}/firehol-defaults.conf" || exit 1
 fi
 
+marksreset() { :; }
+markdef() { :; }
+if [ -r "${FIREQOS_CONFIG_DIR}/fireqos-defaults.conf" ]
+then
+	source "${FIREQOS_CONFIG_DIR}/fireqos-defaults.conf" || exit 1
+fi
+
 common_load_commands $PROGRAM_FILE @AUTOCONF_RUN@ <<-!
 N|TPUT_CMD|@TPUT@|tput
 Y|IP_CMD|@IP@|ip
